index.bs

```diff
@@ -1073,10 +1069,10 @@ run the following steps:
 
 To <dfn>create a {{CookieListItem}}</dfn> from |cookie|, run the following steps.
 
-1. Let |name| be |cookie|'s [=cookie/name=] ([=decoded=]).
-1. Let |value| be |cookie|'s [=cookie/value=] ([=decoded=]).
-1. Let |domain| be |cookie|'s [=cookie/domain=] ([=decoded=]).
-1. Let |path| be |cookie|'s [=cookie/path=] ([=decoded=]).
+1. Let |name| be the result of running [=UTF-8 decode without BOM=] on |cookie|'s [=cookie/name=].
+1. Let |value| be the result of running [=UTF-8 decode without BOM=] on |cookie|'s [=cookie/value=].
+1. Let |domain| be the result of running [=UTF-8 decode without BOM=] on |cookie|'s [=cookie/domain=].
+1. Let |path| be the result of running [=UTF-8 decode without BOM=] on |cookie|'s [=cookie/path=].
```

Comment:
Can these two contain non-ASCII?

Reply:
Per https://httpwg.org/http-extensions/draft-ietf-httpbis-rfc6265bis.html#name-syntax looks like everything is a SHOULD NOT not a MUST NOT. Given that:

* Path looks constrained to any U+0020...U+007E except `;`.  So ASCII only.
* Domain similarly looks constrained to ASCII given the notes. 

I guess we could use `ByteString` here?

Reply:
Yeah, though you'd need additional validation on input either way.